.concourse/kubecf-pool-reconciler.yaml.gomplate

```diff
@@ -1,3 +1,5 @@
+{{ $kind_pool_size := 2 }}
```

Comment:
If changes looks good, we will bump this to the real number of available VMs when we will replace the current pipeline